changelog/unreleased/4507

```diff
@@ -0,0 +1,7 @@
+Enhancement: Upgraded AGP version to 8.7.2
+
+The Android Gradle Plugin version was upgraded to 8.7.2, together with Gradle version (updated to 8.9) and
```

Comment:
```suggestion
The Android Gradle Plugin version has been updated to 8.7.2, together with Gradle version (updated to 8.9) and
```
This should be in present perfect passive, right?

Reply:
Done